lib/ain-ocean/src/api/governance.rs

```diff
@@ -41,30 +42,36 @@ async fn list_gov_proposals(
     let opts = ListProposalsOptions {
         pagination: Some(ListProposalsPagination {
             limit: Some(size),
+            start: query.pagination.next.clone(),
             ..ListProposalsPagination::default()
         }),
         status: query.status,
         r#type: query.r#type,
         cycle: query.cycle,
     };
     let proposals = ctx.client.list_gov_proposals(Some(opts)).await?;
-
-    Ok(ApiPagedResponse::of(proposals, size, |proposal| {
-        proposal.proposal_id.to_string()
-    }))
+    let mut proposals_with_string_amount: Vec<ApiProposalInfo> =
+        proposals.into_iter().map(ApiProposalInfo::from).collect();
+    // proposals.sort_by(|a, b| a.creation_height.cmp(&b.creation_height));
+    proposals_with_string_amount.sort_by(|a, b| a.creation_height.cmp(&b.creation_height));
```

Comment:
hmm.. is sort needed here??

Reply:
if I dont use sort, test case is fail bcoz, default from `rpc` is descending.